column/api/backend/cache/local_mem.py

```diff
@@ -0,0 +1,51 @@
+from collections import deque  # noqa
+import threading
+
+import store
+
+
+# Shared dictionaries for LocalMemoryCache
+_stores = {}
+_key_queues = {}
+_locks = {}
+
+MAX_STORE_SIZE = 50
```

Comment:
Why is there a MAX_STORE_SIZE in both column/api/backend/cache/local_mem.py and column/api/backend/cache/store.py?